expression/builtin_json.go

```diff
@@ -720,7 +723,17 @@ type jsonMergePreserveFunctionClass struct {
 }
 
 func (c *jsonMergePreserveFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
-	return nil, errFunctionNotExists.GenWithStackByArgs("FUNCTION", "JSON_MERGE_PRESERVE")
+	if err := c.verifyArgs(args); err != nil {
+		return nil, err
+	}
+	argTps := make([]types.EvalType, 0, len(args))
+	for range args {
+		argTps = append(argTps, types.ETJson)
+	}
+	bf := newBaseBuiltinFuncWithTp(ctx, args, types.ETJson, argTps...)
+	sig := &builtinJSONMergeSig{bf}
```

Comment:
It will also raise warnings for `json_merge_preseve` in https://github.com/pingcap/tidb/pull/8931/files#diff-01317024270cd2849b5091da7ffc54a1R408?

Reply:
It seems so. So what do you suggest I should do? Removing the warning? Cause I see there are still some deprecated expressions without any warnings in the codebase.

Reply:
What about adding a field to indicate if it is built from `json_merge_preserve` or just check it's pb_code to differentiate them?

Reply:
ok, PTAL